safenode/src/storage/registers.rs

```diff
@@ -292,6 +271,27 @@ impl RegisterStorage {
 
         Ok(stored_reg)
     }
+
+    #[cfg(test)]
+    async fn addrs(&self) -> Vec<RegisterAddress> {
+        self.register_store.addrs().await
+    }
+
+    /// Used for replication of data to new nodes.
+    // Currently only used by the tests, to be used by replication logic
```

Comment:
/// as well?
and here actually shall be a TODO point ?

Reply:
Uhm, not sure, this is old stuff pulled in from old repo.
Will have to check what you mean.

Reply:
Ah, I got it :+1:, added to follow up PR.